webflow-hacks/form-hacks/1.ts

```diff
@@ -1,40 +1,41 @@
 // when the DOM is ready
```

Comment:
This hack has also been written by @PeterEbili in his PR. We need to make sure we coordinate the efforts to avoid situations like this one.

Reply:
I was the one who wrote this. It was the first hack to be merged by @erikmejias to the fs-hacks-rewrite branch, @PeterEbili must have modified the brach.
I added this branch because of changes you suggested...